src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -229,7 +230,7 @@ export default class PickEntityOrConversionDialog extends Vue {
   }
 
   chooseType (entity: SelectOptionsI) {
-    if (entity.value === EntityType.SO || entity.value === EntityType.XSO) {
+    if (!this.isSupportSociety() && (entity.value === EntityType.SO || entity.value === EntityType.XSO)) {
```

Comment:
Please add a short comment to explain the business logic here (eg, why not support society?).